RecoTracker/MkFitCore/src/CandCloner.h

```diff
@@ -0,0 +1,169 @@
+#ifndef RecoTracker_MkFitCore_src_CandCloner_h
+#define RecoTracker_MkFitCore_src_CandCloner_h
+
+#include "MkFinder.h"
+
+#include <vector>
+
+namespace mkfit {
+
+  class IterationParams;
+  class EventOfCombCandidates;
+
+  //#define CC_TIME_LAYER
+  //#define CC_TIME_ETA
+
+  typedef std::pair<int, int> CandClonerWork_t;
+
+  class CandCloner {
+  public:
+    // Maximum number of seeds processed in one call to ProcessSeedRange()
+    static const int s_max_seed_range = MPT_SIZE;
+
+  private:
+    // Temporaries in ProcessSeedRange(), resized/reserved  in constructor.
+
+    // Size of this one is s_max_seed_range
+    std::vector<std::vector<TrackCand>> t_cands_for_next_lay;
+
+  public:
```

Comment:
Each class may have only one each of public, protected, and private sections, which must be declared in that order. (*)

https://cms-sw.github.io/cms_coding_rules.html

Reply:
Addressed in Round 2 follow-up PR trackreco/cmssw#76 commit https://github.com/trackreco/cmssw/pull/76/commits/6f728696277e15a8502cb9e080061ad7d504f604